### PR TITLE
fix(mount): help display format

### PR DESCRIPTION
**Pull request by rolysr**
This change improves readability of current
mount possible options.

### CODE REVIEW

**Comment by uristdwarf** [comment]
Please follow this https://www.conventionalcommits.org/en/v1.0.0/ when making PR's, at least with the PR title, but preferably with the commits as well if we want to rebase. This is to help us automate releases in the future.

File: src/mount/fuse/mount_config.cc [hunk 1]
```diff
@@ -114,28 +114,75 @@ void usage(const char *progname) {
...
 "\n", progname);
-
-	printf("general options:\n");
 	fuse_cmdline_help();
 
 	printf(
-"SFS options:\n"
+"Aditional info options\n"
+"    -o sfsdebug                 print some debugging information\n"
+"\n"
```
Comment (lgsilva3087) [inline src/mount/fuse/mount_config.cc:122]: Consider moving this one to the end. It is usually for debugging and should not be before the General options.

File: src/mount/fuse/mount_config.cc [hunk 2]
```diff
@@ -114,28 +114,75 @@ void usage(const char *progname) {
...
+"Security/permissions related options:\n"
+"    -o sfspassword=PASSWORD     authenticate to sfsmaster with password\n"
+"    -o sfsmd5pass=MD5           authenticate to sfsmaster using directly "
+				"given md5 (only if sfspassword is not defined)\n"
 "    -p   --password             similar to '-o sfspassword=PASSWORD', but "
 				"show prompt and ask user for password\n"
+"    -o askpassword              show prompt and ask user for password\n"
+"    -o sfsdonotrememberpassword do not remember password in memory - more "
+				"secure, but when session is lost then new "
+"\n"
```
Comment (lgsilva3087) [inline src/mount/fuse/mount_config.cc:142]: We need an extra empty line here.

**Review by lgsilva3087 — APPROVED** [review]
Nice change, indeed it looks better. Please see my small comments.

File: src/mount/fuse/mount_config.cc (hunk 1 above)
Comment (rolysr) [reply src/mount/fuse/mount_config.cc]: Done

File: src/mount/fuse/mount_config.cc (hunk 2 above)
Comment (rolysr) [reply src/mount/fuse/mount_config.cc:140]: Done

**Comment by dmga44** [comment]
Nice change, @rolysr. Getting things more similar to our WC :)

Though, I think you should check the rule of the alignment of the options. The description should start at some fixed point, or at least have two spaces between the format of the option and the start of the description. For instance, I think it does not hold for:
- sfsdonotrememberpassword
- cacheexpirationtime

I would also like to not have group of options with a single option, but that is kind of more of my likings.

**Comment by rolysr** [comment]
> * cacheexpirationtime

Done

**Review by dmga44 — APPROVED** [review]

**Pull request merged**